src/routes/api/authors.test.js

```diff
@@ -0,0 +1,147 @@
+const supertest = require('supertest');
+const { format } = require('date-fns');
+const app = require('../../app');
+
+const request = supertest(app.callback());
+
+describe('Author routes', () => {
+  let auth;
+  const userFields = {
+    firstName: 'Test',
+    lastName: 'User',
+    email: 'test@gmail.com',
+    password: 'testPassword',
+  };
+
+  beforeAll(async () => {
+    await app.context.orm.sequelize.sync({ force: true });
+    await app.context.orm.user.create(userFields);
+    const authResponse = await request
+      .post('/api/auth')
+      .set('Content-type', 'application/json')
+      .send({ email: userFields.email, password: userFields.password });
+    auth = authResponse.body;
+  });
+
+  afterAll(async () => {
+    await app.context.orm.sequelize.close();
+  });
+
+  describe('GET /api/authors/:id', () => {
+    let author;
+    let response;
+    const authorData = {
+      firstName: 'John',
+      lastName: 'Doe',
+      birthDate: format(new Date(1993, 2, 22), 'yyyy-MM-dd'),
+    };
+    const authorizedGetAuthor = (id) => request
+      .get(`/api/authors/${id}`)
+      .auth(auth.access_token, { type: 'bearer' });
+    const unauthorizedGetAuthor = (id) => request.get(`/api/authors/${id}`);
+
+    beforeAll(async () => {
+      author = await app.context.orm.author.create(authorData);
+    });
+
+    describe('when passed id corresponds to an existing author', () => {
+      beforeAll(async () => {
+        response = await authorizedGetAuthor(author.id);
+      });
+
+      test('responds with 200 status code', () => {
+        expect(response.status).toBe(200);
+      });
+
+      test('responds with a json body type', () => {
+        expect(response.type).toEqual('application/json');
+      });
+
+      test('body matches snapshot', () => {
+        expect(response.body).toMatchSnapshot();
+      });
+    });
+
+    describe('when passed id does not correspond to any author', () => {
+      test('responds with 404 status code', async () => {
+        response = await authorizedGetAuthor(author.id * -1);
+        expect(response.status).toBe(404);
+      });
+    });
+
+    describe('when request is unauthorized because user is not logged in', () => {
+      test('unauthorized get request to endpoint', async () => {
```

Comment:
Aquí debería ir algo más descriptivo del response, similar al `404`:
```suggestion
      test('responds with 401 status code', async () => {
```